css/style.css

```diff
@@ -61,12 +69,71 @@ input {
   text-align: center;
 }
 
-ul li:nth-child(odd) {
+#booksDisplay ul li:nth-child(odd) {
   background-color: rgb(175, 175, 175);
   color: white;
 }
 
-ul li:nth-child(even) {
+#booksDisplay ul li:nth-child(even) {
   background-color: white;
   color: black;
 }
+
+nav {
+  padding: 0;
+  margin: 0;
+}
+
+.navbar {
+  display: flex;
+  justify-content: flex-end;
+  list-style: none;
+  background-color: rgb(208, 204, 204);
+  padding: 0;
+  margin: 0;
+}
+
+.navbar a {
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+}
```

Comment:
- [ ] [OPTIONAL] For some good UX, consider adding `cursor: pointer` to `links `and `buttons`. On top of this you can add `mouse hover` effects,